newdle/core/util.py

```diff
@@ -89,3 +95,22 @@ def find_overlap(day, start, end, tz):
 def dedent(value, *, _re=re.compile(r'^ +', re.MULTILINE)):
     """Remove leading whitespace from each line."""
     return _re.sub('', value)
+
+
+def render_user_avatar(initials, size):
+    avatar = render_template('avatar.svg', text=initials.upper(), size=size)
+    return send_file(BytesIO(avatar.encode()), mimetype='image/svg+xml')
+
+
+def avatar_payload_from_user_info(user_info):
+    return secure_serializer.dumps(
+        {
+            'email': user_info['email'],
+            'initials': user_info['first_name'][0].upper(),
+        },
+        salt='avatar-payload',
+    )
+
+
+def user_info_from_payload(payload):
```

Comment:
`avatar_info_from_payload`? since it's not really the whole user info, just the stuff relevant to the avatar..